services/w3c/w3c-validation-helper.js

```diff
@@ -51,11 +51,9 @@ const getSchema = preset => {
     } else {
       schema.push('http://s.validator.nu/xhtml10/xhtml-frameset.rnc')
     }
-    schema.push('http://s.validator.nu/html4/assertions.sch')
```

Comment:
According this https://github.com/validator/validator/issues/559 it appears that this is being removed